Sources/LiveViewNative/Coordinators/LiveSessionCoordinator.swift

```diff
@@ -514,24 +512,21 @@ class LiveSessionURLSessionDelegate<R: RootRegistry>: NSObject, URLSessionTaskDe
         guard let url = request.url else {
             return request
         }
-        let components = URLComponents(url: url, resolvingAgainstBaseURL: false)
         
         var newRequest = request
-        if !(components?.queryItems?.contains(where: { $0.name == "_format" }) ?? false) {
-            newRequest.url = url.appending(queryItems: [.init(name: "_format", value: await LiveSessionCoordinator<R>.platform)])
-        }
+        newRequest.url = await url.appendingLiveViewItems(R.self)
         return newRequest
     }
 }
 
 extension LiveSessionCoordinator {
     static var platform: String { "swiftui" }
-    static var platformParams: Payload {
+    static var platformParams: [String:String] {
         [
             "app_version": getAppVersion(),
             "app_build": getAppBuild(),
             "bundle_id": getBundleID(),
-            "format": "swiftui",
+            "format": platform,
```

Comment:
Do we need to pass the format in the `_interface` too? Or can I remove this.

Reply:
it can be removed